incubator/python-flask/image/Dockerfile-stack

```diff
@@ -24,11 +31,10 @@ ENV APPSODY_TEST="python -m unittest discover -s test -p *.py"
 ENV APPSODY_TEST_ON_CHANGE=$APPSODY_TEST
 ENV APPSODY_TEST_KILL=false
 
-COPY ./LICENSE /licenses/
-COPY ./project /project
-COPY ./config /config
-WORKDIR /project
-RUN pipenv install
+COPY --chown=worker:worker ./LICENSE /licenses/
```

Comment:
If we're using the chown flag here, please can you add a requirements field to the `stack.yaml`. I.e.
```
.
.
.
default-template: simple
license: Apache-2.0
requirements:
  docker-version: ">= 17.09.0"
```
This will stop users on older versions of Docker hitting errors after they've already started developing.

Reply:
will do, thx